src/components/ListItem.jsx

```diff
@@ -1,12 +1,22 @@
 import './ListItem.css';
 import { useCallback, useEffect, useState } from 'react';
 import { updateItem } from '../api/firebase';
-import { getFutureDate } from '../utils';
+import {
+	getFutureDate,
+	getDaysBetweenDates,
+	dateHasPassed,
+	purchaseSchedule,
+} from '../utils';
```

Comment:
The `getDaysBetweenDates` and `dateHasPassed` functions, do they still have to be exported/imported here? It looks they are used by `purchaseSchedule` directly but not by the ListItem component.

Reply:
Great observation! We've removed those extra imports